celerdata/assets/logs/celerdata_tests.yaml

```diff
@@ -0,0 +1,30 @@
+id: "celerdata"
+tests:
```

Comment:
It could be great to add the 4 samples from the grok parser here as it will cover all use cases above!

Reply:
Done and thank you for you review work!

Reply:
Great!